c_cxx/MNIST/readme.md

```diff
@@ -0,0 +1 @@
+See [ReadMe.md](../README.md) for build instructions.
```

Comment:
does this need to be run from the same directory that the executable is in? e.g., to find mnist.onnx. might be worth calling that out if so.